README.md

```diff
@@ -12,10 +12,12 @@
 [![ppc64le build status badge](https://img.shields.io/jenkins/s/https/doi-janky.infosiftr.net/job/multiarch/job/ppc64le/job/nextcloud.svg?label=ppc64le)](https://doi-janky.infosiftr.net/job/multiarch/job/ppc64le/job/nextcloud)
 [![s390x build status badge](https://img.shields.io/jenkins/s/https/doi-janky.infosiftr.net/job/multiarch/job/s390x/job/nextcloud.svg?label=s390x)](https://doi-janky.infosiftr.net/job/multiarch/job/s390x/job/nextcloud)
 
-A safe home for all your data. Access & share your files, calendars, contacts, mail & more from any device, on your terms.
+Nextcloud is a safe home for all your data. Access & share your files, calendars, contacts, mail & more from any device, on your terms.
 
 ![logo](https://cdn.rawgit.com/nextcloud/docker/80dd587d847b184ba95d7187a2a7a56ae4cbbb7b/logo.svg)
 
+This Docker micro-service image is developed and maintained by the Nextcloud community. Nextcloud GmbH does not maintain nor support this Docker image. When you are looking to get official support, you can become an [enterprise](https://nextcloud.com/enterprise/) customer or use [AIO](https://github.com/nextcloud/all-in-one#nextcloud-all-in-one) which is a by the Nextcloud GmbH maintained Docker-based project.
```

Comment:
I would just remove `which is a by the Nextcloud GmbH maintained Docker-based project`. Seems redundant.

Reply:
Good idea. Done!